tests/Aspire.Microsoft.Data.SqlClient.Tests/ConformanceTests.cs

```diff
@@ -71,5 +81,26 @@ protected override void SetMetrics(MicrosoftDataSqlClientSettings options, bool
         => options.Metrics = enabled;
 
     protected override void TriggerActivity(SqlConnection service)
-        => service.Open();
```

Comment:
Was just opening a connection not enough to trigger an activity?

Reply:
Yeah, nothing was emitted with just that, and the tests would fail:
```
[xUnit.net 00:00:23.52]     Aspire.Microsoft.Data.SqlClient.Tests.ConformanceTests.TracingEnablesTheRightActivitySource [FAIL]
  Failed Aspire.Microsoft.Data.SqlClient.Tests.ConformanceTests.TracingEnablesTheRightActivitySource [7 s]
  Error Message:
   Microsoft.DotNet.RemoteExecutor.RemoteExecutionException : Remote process failed with an unhandled exception.
  Stack Trace:

Child exception:
  Xunit.Sdk.NotEmptyException: Assert.NotEmpty() Failure
   at Aspire.Components.ConformanceTests.ConformanceTests`2.ActivitySourceTest(String key) in /Users/ankj/dev/a3/tests/Aspire.Components.Common.Tests/ConformanceTests.cs:line 405
   at Aspire.Microsoft.Data.SqlClient.Tests.ConformanceTests.<>c.<TracingEnablesTheRightActivitySource>b__24_1(ConformanceTests obj) in /Users/ankj/dev/a3/tests/Aspire.Microsoft.Data.SqlClient.Tests/ConformanceTests.cs:line 94
   at Aspire.Microsoft.Data.SqlClient.Tests.ConformanceTests.RunWithFixtureAsync(Action`1 test) in /Users/ankj/dev/a3/tests/Aspire.Microsoft.Data.SqlClient.Tests/ConformanceTests.cs:line 104
   at Aspire.Microsoft.Data.SqlClient.Tests.ConformanceTests.RunWithFixtureAsync(Action`1 test) in /Users/ankj/dev/a3/tests/Aspire.Microsoft.Data.SqlClient.Tests/ConformanceTests.cs:line 104
   at Microsoft.DotNet.RemoteExecutor.Program.Main(String[] args) in /_/src/Microsoft.DotNet.RemoteExecutor/src/Program.cs:line 65

Child process:
  Aspire.Microsoft.Data.SqlClient.Tests, Version=42.42.42.42, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51 Aspire.Microsoft.Data.SqlClient.Tests.ConformanceTests+<>c System.Threading.Tasks.Task <TracingEnablesTheRightActivitySource>b__24_0()
```